.travis.yml

```diff
@@ -1,17 +1,17 @@
 language: python
 python:
   - "3.6"
-dist: xenial
```

Comment:
This one was added to make sure travis does not hang again. Are you sure it can be removed?

Reply:
`xenial` is the default now.

Reply:
Hope they won't change it soon ;)